TodoList/src/app/screens/BookList/index.js

```diff
@@ -5,7 +5,16 @@ import books from '../../../constants/books';
 import Book from './components/Book';
 
 class BookList extends Component {
-  renderItem = ({ item }) => <Book image={item.image_url} title={item.title} author={item.author} key={item.id} />;
+  renderItem = ({ item }) => (
+    <Book
+      image={item.image_url}
+      title={item.title}
+      author={item.author}
+      key={item.id}
+      navigation={this.props.navigation}
+      id={item.id}
```

Comment:
You don't need to pass down the `id`. But, since you will need to show most of the book on the `BookDetails` component, maybe it´s a good idea to pass down the entire book here.